about/faq.rst

```diff
@@ -48,74 +39,51 @@ Which platforms are supported by Godot?
-C++ binders). None of them worked as well as GDScript does now.
-
-More information about getting comfortable with GDScript or dynamically
-typed languages can be found in the :ref:`doc_gdscript_more_efficiently`
-tutorial.
-
-For the more technically versed, proceed to the next item.
-
-I don't believe you. What are the technical reasons for the item above?
+GDScript is Godot's integrated scripting language. It was built from the ground up to maximize Godot's potential in the least amount of code, affording both novice and expert developers alike to capitalize on Godot's strengths as fast as possible. If you've ever written anything in a language like Python before then you'll feel right at home. For examples, history, and a complete overview of the power GDScript offers you, check out the `GDScript scripting guide <gdscript_basics>`_.
+
+There are several reasons to use GDScript--especially when you are prototyping, in alpha/beta stages of your project, or are not creating the next AAA title--but the most salient reason is the overall **reduction of complexity.** 
+
+The original intent of creating a tightly integrated, custom scripting language for Godot was two-fold: first, it reduces the amount of time necessary to get up and running with Godot, giving developers a rapid way of exposing themselves to the engine with a focus on productivity; second, it reduces the overall burden of maintenance, attenuates the dimensionality of issues, and allows the developers of the engine to focus on squashing bugs and improving features related to the engine core--rather than spending a lot of time trying to get a small set of incremental features working across a large set of languages. 
+
+Since Godot is an open-source project, it was imperative from the start to prioritize a more integrated and seamless experience over attracting additional users by supporting more familiar programming languages--especially when supporting those more familiar languages would result in a worse experience. We understand if you would rather use another language in Godot (see list of supported options above). That being said, if you haven't given GDScript a try, try it for **three days**. Just like Godot, once you see how powerful it is and rapid your development becomes, we think GDScript will grow on you. 
+
+More information about getting comfortable with GDScript or dynamically typed languages can be found in the :ref:`doc_gdscript_more_efficiently` tutorial.
+
+What were the motivations behind creating GDScript?
 -----------------------------------------------------------------------
 
-The main reasons are:
+The main reasons for creating a custom scripting language for Godot were:
 
 1. No good thread support in most script VMs, and Godot uses threads
    (Lua, Python, Squirrel, JS, AS, etc.).
-2. No good class extending support in most script VMs, and adapting to
+2. No good class-extending support in most script VMs, and adapting to
```

Comment:
Again "Poor" or "Lack of".

Reply:
I think I'm going to go with "poor" for all of these. I know it's just semantics, but I don't know the original reasoning from the devs but as a dev myself I can only assume that there *was* support, but that it wasn't adequate for what they were doing.